scripts/variantstore/wdl/extract/hail_create_vat_inputs.py

```diff
@@ -54,7 +54,8 @@ def failing_gts_to_no_call(vds):
      FT is False => GT assigned no-call
      FT is missing => GT keeps its current value
     """
-    vd = vds.variant_data
+    gt_vds = replace_lgt_with_gt(vds)
+    vd = gt_vds.variant_data
```

Comment:
looks like line 13 should be unsquiggled and otherwise made consistent with these changes